capstone-ui/src/components/App/App.jsx

```diff
@@ -11,19 +9,21 @@ import Login from "../Login/Login";
 import { useState } from "react";
 import axios from "axios";
 import SignUp from "../SignUp/SignUp";
+import Home from "../Home/Home";
+import Contact from "../Contact/Contact";
+import Footer from "../Footer/Footer";
+import Faqs from "../FAQs/FAQs";
+import SuccesStories from "../SuccessStories/SuccessStories";
+import ProfileCard from "../ProfileCard/ProfileCard";
+import Parse from "parse/dist/parse.min.js";
```

Comment:
Are we using Parse in this file? 
Does the app works the same if we remove this line?

Reply:
was using Parse to test and forgot to remove after test